### DISCUSSION TITLE
EAC style logfile

### DISCUSSION THREAD

**Discussion by zordaz** [enzo1982/freac#410] [Your Ideas]
Feature request:
Implement an option in Fre:ac to generate an Exact Audio Copy style logfile after ripping a CD.

**Comment by enzo1982** [comment]
Log file generation is already implemented and can be configured in the configuration dialog under _Extensions->Logging_.

**Reply by zordaz** [reply]
> Log file generation is already implemented and can be configured in the configuration dialog under _Extensions->Logging_.

Yes, I know. But with EAC style logging I also mean the lay out/structure of the logfile. CUETools/CUERipper for example has the option the create logging in EAC style or in CUETools style.
